specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-09-02-preview/examples/Fleets_CreateOrUpdate.json

```diff
@@ -0,0 +1,90 @@
+{
+  "parameters": {
+    "api-version": "2022-09-02-preview",
+    "subscriptionId": "subid1",
+    "resourceGroupName": "rg1",
+    "fleetName": "fleet-1",
+    "parameters": {
+      "tags": {
+        "tier": "production",
+        "archv2": ""
+      },
+      "location": "East US",
+      "properties": {
+        "hubProfile": {
+          "dnsPrefix": "dnsprefix1"
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "headers": {
+        "ETag": "23ujdflewrj3=",
+        "Location": "https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myResourceGroup/providers/Microsoft.ContainerService/locations/eastus/operations/00000000-0000-0000-0000-000000000000?api-version=2022-02-01",
+        "Azure-AsyncOperation": "https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myResourceGroup/providers/Microsoft.ContainerService/locations/eastus/operationResults/00000000-0000-0000-0000-000000000000?api-version=2022-02-01"
+      },
+      "body": {
+        "id": "/subscriptions/subid1/resourcegroups/rg1/providers/Microsoft.ContainerService/fleets/fleet-1",
+        "name": "fleet-1",
+        "type": "Microsoft.ContainerService/fleets",
+        "systemData": {
+          "createdBy": "someUser",
+          "createdByType": "User",
+          "createdAt": "2022-03-23T05:40:40.657Z",
+          "lastModifiedBy": "someOtherUser",
+          "lastModifiedByType": "User",
+          "lastModifiedAt": "2022-03-23T05:40:40.657Z"
+        },
+        "tags": {
+          "tier": "production",
+          "archv2": ""
+        },
+        "location": "East US",
+        "etag": "23ujdflewrj3=",
+        "properties": {
+          "hubProfile": {
+            "dnsPrefix": "dnsprefix1",
+            "fqdn": "dnsprefix1-abcd1234.flt.eastus.azmk8s.io",
+            "kubernetesVersion": "1.22.4"
+          },
+          "provisioningState": "Updating"
+        }
+      }
+    },
+    "201": {
+      "headers": {
+        "ETag": "23ujdflewrj3=",
+        "Location": "https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myResourceGroup/providers/Microsoft.ContainerService/locations/eastus/operations/00000000-0000-0000-0000-000000000000?api-version=2022-02-01",
```

Comment:
>Location [](http://example.com/codeflow?start=9&length=8)

I thought Location is supposed to be the URL of the created resource according to HTTP